addons/survey/tests/test_survey_invite.py

```diff
@@ -135,6 +136,21 @@ def test_survey_invite_authentication_signup(self):
             set([self.customer.email, self.user_emp.email, self.user_portal.email]))
         self.assertEqual(answers.mapped('partner_id'), self.customer | self.user_emp.partner_id | self.user_portal.partner_id)
 
+    @users('survey_manager')
+    def test_survey_invite_email_from(self):
+        # Verifies whether changing the value of the "email_from" field reflects on the receiving end.
+        action = self.survey.action_send_survey()
+        invite_form = Form(self.env[action['res_model']].with_context(action['context']))
+        invite_form.partner_ids.add(self.user_portal.partner_id)
+        invite_form.template_id.write({'email_from':self.survey_user.email_formatted})
```

Comment:
I was proposing to use an inline template enabled input, like '{{ object.survey_id.user_id.email_formatted }}', to test that rendering is effectively done (and not just copy pasting the template value).